### ISSUE TITLE
Elevated circulating methylmalonic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6013]
**Preferred term label:** Elevated circulating methylmalonic acid concentration

**Synonyms** Methylmalonic Acidemia

**Definition (free text, please give PubMed ID)** Concentration of methylmalonic acid in the blood is above the normal range., PMID: 26101005

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This term already exists, https://hpo.jax.org/app/browse/term/HP:0002912
I will add the synonym
